pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -242,9 +242,20 @@ func (ssr *Snapshotter) stop() {
 		ssr.deltaSnapshotTimer = nil
 	}
 	ssr.closeEtcdClient()
+}
 
+// SetSnapshotterInactive set the snapshotter state to Inactive.
+func (ssr *Snapshotter) SetSnapshotterInactive() {
+	ssr.SsrStateMutex.Lock()
+	defer ssr.SsrStateMutex.Unlock()
 	ssr.SsrState = brtypes.SnapshotterInactive
-	ssr.SsrStateMutex.Unlock()
+}
+
+// SetSnapshotterActive set the snapshotter state to active.
+func (ssr *Snapshotter) SetSnapshotterActive() {
```

Comment:
Who is calling this?

Reply:
If this method is not used then please remove it as it is dead code.